src/pages/index.js

```diff
@@ -81,35 +143,174 @@ profileEditButton.addEventListener("click", () => {
 const imagePopup = new PopupWithImage("#preview-modal");
 imagePopup.setEventListeners();
 
-// Enable form validation
+// add event listener to avatar
+document
+  .querySelector(".profile__edit-avatar-button")
+  .addEventListener("click", () => {
+    editAvatarModal.open();
+  });
+// when it fires, open avatar image modal
+
+/* ------------------------ Enable form validation ------------------------ */
+
+const editFormValidator = new FormValidator(
+  validationSettings,
+  profileEditForm
+);
+const addFormValidator = new FormValidator(
+  validationSettings,
+  addCardFormElement
+);
+
+const editAvatarValidator = new FormValidator(
+  validationSettings,
+  document.querySelector("#edit-avatar-modal")
```

Comment:
`  document.querySelector("#edit-avatar-modal")
`

It's a `modal`. but you need to pass a `form` here